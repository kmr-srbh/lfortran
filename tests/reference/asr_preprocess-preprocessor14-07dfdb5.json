{
    "basename": "asr_preprocess-preprocessor14-07dfdb5",
    "cmd": "lfortran --cpp --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/preprocessor14.f90",
    "infile_hash": "ecfd32142988c081654be18462888b79f4d47174d63c9e2b37b5a3d1",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr_preprocess-preprocessor14-07dfdb5.stdout",
    "stdout_hash": "c4038825acd5268f9ac65ae1292e0eaf7f00b15055e2d800385690c9",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}