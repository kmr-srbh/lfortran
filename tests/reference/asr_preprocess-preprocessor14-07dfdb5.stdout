(TranslationUnit
    (SymbolTable
        1
        {
            preprocessor14:
                (Program
                    (SymbolTable
                        2
                        {
                            
                        })
                    preprocessor14
                    []
                    [(Print
                        [(StringConstant
                            "X == 1"
                            (Character 1 6 () PointerString)
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringConstant
                            "X == 1"
                            (Character 1 6 () PointerString)
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringConstant
                            "X == 1"
                            (Character 1 6 () PointerString)
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringConstant
                            "X == 1"
                            (Character 1 6 () PointerString)
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringConstant
                            "X == 1"
                            (Character 1 6 () PointerString)
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringConstant
                            "X == 1"
                            (Character 1 6 () PointerString)
                        )]
                        ()
                        ()
                    )]
                )
        })
    []
)
